portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -1719,30 +1794,249 @@ definition {
 	}
 
 	@description = "LPS-124960 When there's a WebContent1 authored by Author1 and has Category1 and Category2 and a WebContent2 authored by Author2 and has Category1, and the user filters by Author1 and Category1, only the WebContent1 is shown in the list"
-	@ignore = "true"
 	@priority = "3"
 	test FilterByCategoryAndAuthor {
```

Comment:
Add property custom.properties = "jsonws.web.service.paths.excludes="; to enable JSONUser.setFirstPassword